psychopy/data.py

```diff
@@ -3464,8 +3464,8 @@ class QuestHandler(StairHandler):
         # create staircase object
         # trying to find out the point where subject's response is 50 / 50
         # if wanted to do a 2AFC then the defaults for pThreshold and gamma
-        # are good
-        staircase = data.QuestHandler(staircase._nextIntensity, 0.2,
+        # are good. As start value, we'll use 50% contrast, with SD = 20%
+        staircase = data.QuestHandler(0.5, 0.2,
```

Comment:
So I think I figured it out. The priors are ordinary gaussians (normal distributions). While QUEST uses Weibull functions for its computations.

The normal distributions are normalized "manually"
```python
        self.pdf = num.exp(-0.5*(self.x/self.tGuessSd)**2)
        self.pdf = self.pdf/num.sum(self.pdf)
```
therefore the formula looks a bit different than in text books, i.e. no 1/sqrt(2 pi sigma^2) normalization term; instead, a two-step process to normalization. :)